snapshotTestContent/SharedMatrixDeletes/current_snapshots/snapshot_457_0.json

```diff
@@ -1874,7 +1874,7 @@
                                             "path": "header",
                                             "type": "Blob",
                                             "value": {
-                                              "contents": "{\"version\":\"1\",\"segmentCount\":8,\"length\":8,\"segments\":[[1,4],{\"json\":[1,1],\"removedSeq\":225,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"},{\"json\":[1,2],\"removedSeq\":222,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"},{\"json\":[1,3],\"removedSeq\":216,\"removedClient\":\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\"},{\"json\":[1,5],\"removedSeq\":207,\"removedClient\":\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\"},{\"json\":[1,6],\"removedSeq\":204,\"removedClient\":\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\"},{\"json\":[1,7],\"removedSeq\":195,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"},{\"json\":[1,8],\"removedSeq\":192,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"}],\"startIndex\":0,\"headerMetadata\":{\"minSequenceNumber\":189,\"sequenceNumber\":457,\"orderedChunkMetadata\":[{\"id\":\"header\"}],\"totalLength\":8,\"totalSegmentCount\":8}}",
+                                              "contents": "{\"version\":\"1\",\"segmentCount\":8,\"length\":8,\"segments\":[[1,4],{\"json\":[1,1],\"removedSeq\":225,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\",\"removedClientIds\":[\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"]},{\"json\":[1,2],\"removedSeq\":222,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\",\"removedClientIds\":[\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"]},{\"json\":[1,3],\"removedSeq\":216,\"removedClient\":\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\",\"removedClientIds\":[\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\",\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"]},{\"json\":[1,5],\"removedSeq\":207,\"removedClient\":\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\",\"removedClientIds\":[\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\",\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"]},{\"json\":[1,6],\"removedSeq\":204,\"removedClient\":\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\",\"removedClientIds\":[\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\",\"5f9c6a07-e72b-4176-95c2-2683be590ee2\"]},{\"json\":[1,7],\"removedSeq\":195,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\",\"removedClientIds\":[\"5f9c6a07-e72b-4176-95c2-2683be590ee2\",\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\"]},{\"json\":[1,8],\"removedSeq\":192,\"removedClient\":\"5f9c6a07-e72b-4176-95c2-2683be590ee2\",\"removedClientIds\":[\"5f9c6a07-e72b-4176-95c2-2683be590ee2\",\"cec2de69-45a7-4a6b-9baf-c0ef0c3f0860\"]}],\"startIndex\":0,\"headerMetadata\":{\"minSequenceNumber\":189,\"sequenceNumber\":457,\"orderedChunkMetadata\":[{\"id\":\"header\"}],\"totalLength\":8,\"totalSegmentCount\":8}}",
```

Comment:
Why are the contents here bigger?

Reply:
All these changes are in "SharedMatrixDeletes" which is what was added after the bug was introduced - https://dev.azure.com/fluidframework/internal/_workitems/edit/233